R/get_functions.R

```diff
@@ -133,3 +133,43 @@ get_xml <- function(path, textfield, encoding,...) {
 }
 
 
+get_html <- function(f, ...) {
+    args <- list(...)
+
+    # http://stackoverflow.com/a/3195926
+    html <- XML::htmlTreeParse(f, useInternal = TRUE)
+    txt <- XML::xpathApply(html, "//body//text()[not(ancestor::script)][not(ancestor::style)][not(ancestor::noscript)]", 
+                           XML::xmlValue)
+    txt <- txt[!grepl('^\\s*$', txt)] # Remove text which is just whitespace
+    txt <- paste0(txt, collapse='')
+
+    data.frame(texts = txt, stringsAsFactors = FALSE)
+}
+
+
+get_pdf <- function(f, ...) {
+    args <- list(...)
+
+    txt <- pdftools::pdf_text(f)
```

Comment:
Nice, I had not known about this package. Looks portable and is probably fast too. I was thinking we would require the user to install xpdf but this is better.